.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "ZPascal"
```

Comment:
@chombium What do you think, should we add a team here?

Reply:
I don't think that we need a team at the moment. I think that for the time being we can leave it as a list of users.